orderer/common/channelparticipation/restapi.go

```diff
@@ -70,13 +70,84 @@ func NewHTTPHandler(config localconfig.ChannelParticipation, registrar ChannelMa
 		router:    mux.NewRouter(),
 	}
 
+	// swagger:operation GET /channels/{channelID} channels listChannel
+	// ---
+	// summary: Returns detailed channel information for an Ordering Service Node (OSN).
+	// parameters:
+	// - name: channelID
+	//   in: path
+	//   description: Channel ID
+	//   required: true
+	//   type: string
+	// responses:
+	//    "200":
+	//       description: Successfully retrieved channel.
+	//       schema:
+	//         "$ref": "#/definitions/channelInfo"
+
 	handler.router.HandleFunc(urlWithChannelIDKey, handler.serveListOne).Methods(http.MethodGet)
 
+	// swagger:operation DELETE /channels/{channelID} channels deleteChannel
+	// ---
+	// summary: The OSN is removed from the channel via a config transaction.
+	// parameters:
+	// - name: channelID
+	//   in: path
+	//   description: Channel ID
+	//   required: true
+	//   type: string
+	// responses:
+	//    '204':
+	//      description: Successfully removed channel.
+	//    '400':
+	//      description: Bad request.
+	//    '404':
+	//      description: The channel does not exist.
+	//    '405':
+	//      description: The system channel exists, removal is not allowed.
+	//    '409':
+	//      description: The channel is pending removal.
+
 	handler.router.HandleFunc(urlWithChannelIDKey, handler.serveRemove).Methods(http.MethodDelete)
 	handler.router.HandleFunc(urlWithChannelIDKey, handler.serveNotAllowed)
 
+	// swagger:operation GET /channels channels listChannels
+	// ---
+	// summary: Provides a list channel information for an Ordering Service Node (OSN).
+	// responses:
+	//    "200":
+	//       description: Successfully retrieved channel.
+	//       schema:
+	//         "$ref": "#/definitions/channelList"
+
 	handler.router.HandleFunc(URLBaseV1Channels, handler.serveListAll).Methods(http.MethodGet)
 
+	// swagger:operation POST /channels channels createChannel
```

Comment:
```suggestion
	// swagger:operation POST /channels channels joinChannel
```

Reply:
fyi this is only a operationID used to identify each unique operation but otherwise this is only naming 🧹